.github/workflows/cd-deploy.yml

```
@@ -1,458 +1,458 @@
-name: CD - Deploy
+# name: CD - Deploy
```

actionlint (1.7.8)

workflow is empty

(syntax-check)